pkg/render/applicationlayer.go

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
```

Comment:
We're moving towards a different subpackage architecture, so this should be in an "applicationlayer" subpackage instead of at the root of the render folder.

Look at the logstorage, kubecontrollers, and intrusiondetection packages in the render folder for examples of what I mean. Also DM me if there's anything you want to talk about.

Reply:
refactored it in d17bee13c2976396b9f6a3e918c031a77eae8606